### PR TITLE
Updating database generation scripts

### PR DESCRIPTION
**Pull request by dr-rodriguez**
Main changes:

- Updating database generation script to produce either a sqlite binary file or load to a postgres database
- Update the Github-Actions script to generate both databases, loading the postgres one to a Heroku instance. This may allow us to completely avoid generating the binary file and dealing with pushing to a separate repo (ie, #163).

### CODE REVIEW

File: scripts/tutorials/generate_database.py [hunk 1]
```diff
@@ -38,14 +31,48 @@
...
+                connection_string = sys.argv[2]
+            else:
+                connection_string = os.getenv('SIMPLE_DATABASE_URL')
+            load_postgres(connection_string)
+        elif architecture == 'sqlite':
+            load_sqlite()
+        else:
+            print(f'Unrecognized architecture: {architecture}')
+    else:
+        print('Specify database architecture (eg, postgres, sqlite)')
```
Comment (Will-Cooper) [inline scripts/tutorials/generate_database.py:78]: Not a huge fan of this implementation, you could do this with ```argparse```, no?

**Review by Will-Cooper — CHANGES_REQUESTED** [review]
LGTM, will be curious to see if this fixes #163

**Comment by Will-Cooper** [comment]
Also let me know if we'll also need to update SIMPLE-web to match with the ```wget```

File: scripts/tutorials/generate_database.py (hunk 1 above)
Comment (dr-rodriguez) [reply scripts/tutorials/generate_database.py]: This was quick and dirty, but looking through `argparse` I like how it works so I'll update it.

**Review by Will-Cooper — APPROVED** [review]
LGTM, I like the ```argparse``` change for sure

**Pull request merged**